qgreenland/config/layers/Basemaps/Internet required/online.py

```diff
@@ -0,0 +1,33 @@
+from qgreenland.config.datasets.online import gibs
+from qgreenland.models.config.layer import Layer, LayerInput
+
+blue_marble_layer = Layer(
+    id="blue_marble_shaded_relief_bathymetry",
+    title="Blue Marble shaded relief and Bathymetry (500m)",
```

Comment:
```suggestion
    title="Blue Marble shaded relief and bathymetry (500m)",
```

Just noticed we used capitalization unnecessarily here. Accepting this change will trigger the need for a config lock though...

Reply:
ba48aeab